src/main/kotlin/services/AssignmentService.kt

```diff
@@ -12,7 +11,7 @@ import org.springframework.data.domain.Page
 import org.springframework.data.domain.Pageable
 import org.springframework.stereotype.Service
 import java.time.LocalDate
-import java.util.UUID
```

Comment:
Hmmm.. I think you want to change your intellij settings to not do wildcard imports (often times java.util.* are mentioned in the excluded one). Soon ktlint will make your build fail on wildcards

Reply:
Tnx, fixed